Source/Fuse.Controls.Primitives/Docs/Text/Examples.md

```diff
@@ -1,4 +1,4 @@
-# Examples
+# Examples 
```

Comment:
Nit: extra whitespace here; less of a problem in docs than code due to git blame info being clobbered, but try to look out for that :)